1/Dockerfile

```diff
@@ -48,6 +48,35 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/subversion.jpi.pinned && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
+    chmod -R 777 /etc/alternatives && \
+    chmod -R 777 /var/lib/alternatives && \
+    chmod -R 777 /usr/lib/jvm && \
+    chmod 777 /usr/bin && \
+    chmod 777 /usr/lib/jvm-exports && \
+    chmod 777 /usr/share/man/man1 && \
```

Comment:
OK ... switching to 775 did in fact work for me when running the tests.  Must have misinterpreted something earlier.

Pushed the updates.